docs/guides/cookies.md

```diff
@@ -70,6 +74,42 @@ console.log(await context.cookies()); // print available cookies.
 
 [Browser.deleteCookie()](https://pptr.dev/api/puppeteer.browser.deletecookie) method allows deleting cookies from storage.
 
+```ts
+import puppeteer from 'puppeteer';
+
+const browser = await puppeteer.launch();
+
+const context = await browser.createBrowserContext();
+
+// Deletes two cookies for the localhost domain.
+await browser.deleteCookie(
+  {
+    name: 'cookie1',
+    value: '1',
+    domain: 'localhost',
+    path: '/',
+    sameParty: false,
+    expires: -1,
+    httpOnly: false,
+    secure: false,
+    sourceScheme: 'NonSecure',
+  },
+  {
+    name: 'cookie2',
+    value: '2',
+    domain: 'localhost',
+    path: '/',
+    sameParty: false,
+    expires: -1,
+    httpOnly: false,
+    secure: false,
+    sourceScheme: 'NonSecure',
+  },
+);
+
+console.log(await context.cookies()); // print available cookies.
```

Comment:
```suggestion
```ts
import puppeteer from 'puppeteer';

const browser = await puppeteer.launch();

// Deletes two cookies for the localhost domain.
await browser.deleteCookie(
  {
    name: 'cookie1',
    value: '1',
    domain: 'localhost',
    path: '/',
    sameParty: false,
    expires: -1,
    httpOnly: false,
    secure: false,
    sourceScheme: 'NonSecure',
  },
  {
    name: 'cookie2',
    value: '2',
    domain: 'localhost',
    path: '/',
    sameParty: false,
    expires: -1,
    httpOnly: false,
    secure: false,
    sourceScheme: 'NonSecure',
  },
);

console.log(await browser.cookies()); // print available cookies.
```